### ISSUE TITLE
enable windows 64 bit arm support

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13914]
### Describe the solution you'd like
I would like to ask if nvda is not adapted yet.
Allow 64-bit windows support on arm platform.
I don't have an arm machine, but I believe this is the future.
I confess that what made me think about opening this call is the news on the scientific freedoms page of jaws beta support for the arm 64bits platform.
I believe it is important for nvda to adapt to new technologies, even though I don't have them.
Screen reader support should be possible due to the previously mentioned ad, it is necessary to know how to adapt the screen reader.

**Comment by josephsl** [comment]
Hi,

While not native yet, NVDA supports Windows 10 and 11 on ARM64. In order to go native (full ARM64 capability), ARM64 Python must be used. But that's just the tip of the iceberg:

1. 64-bit x86 (x64 in Microsoft documentation) and ARM64 cannot understand each other's machine code. This is why an emulator is used on Windows on ARM to run 32-bit x86 programs such as NVDA. Windows 11 brought ARM64EC, an ability to run 64-bit x86 code on ARM processors, but in order to do this, NVDA must first be ported to use pure 64-bit x86 Python. This requires altering significant parts of NVDA, most notably speech synthesizer support and working with 32-bit apps and DLL's.
2. Add-ons: some add-ons (notably, speech synthesizers) come with DLL's and pyd files, both of which are 32-bit. In order to run these add-ons on ARM64 natively, two emulation layers or communication channels must be used: a channel to facilitate communication between 32-bit and 64-bit x86 machine code (this is how NVDA can support 64-bit apps in some scenarios now), and another channel to let ARM64 components talk to x86 machine code. Therefore, add-on authors wishing to support 32-bit x86, 64-bit x86, and 64-bit ARM must bundle up to three DLL and pyd files, one per architecture if one wants to go all native.

IN some ways, ever since NVDA added support for Windows on ARM64 in 2017, I anticipated this suggestion. To prepare for this, some of my own add-ons (notably, Resource Monitor) were edited to at least detect the three architectures listed above.

Hope this helps. Thanks.

**Comment by seanbudd** [comment]
Closing this issue as NVDA currently supports ARM 64 bit.
